utils/merge_and_push_translations.sh

```diff
@@ -79,64 +147,36 @@ if [ -n "$(git status -s)" ]; then
   # Push source catalogs to Transifex
   tx push -s
   sleep 65 # wait for translation files to be updated
-fi
 
-# Pull All translations from Transifex
-# tx pull -a
-# Force-pull translations because some would get skipped accidentally
-tx pull -fa
-
-# Update authors file
-python3 utils/update_authors.py
-if [ $? -eq 0 ] ; then
-  echo "Updated authors";
-else
-  echo "Failed updating authors";
-  exit 1;
-fi
+  # Pull All translations from Transifex
+  # use force to make sure really all files get pulled
+  tx pull -a -f
 
-# Update appdata
-python3 utils/update_appdata.py
-if [ $? -eq 0 ] ; then
-  echo "Updated appdata";
-else
-  echo "Failed updating appdata";
-  exit 1;
+  # Undo one-liner diffs of pure timestamps with no other content
+  undo_oneliner_diffs
```

Comment:
Hmm, I believe this shouldn't be necessary any more, but tx tricked me last time too, so let's see it in action.

Reply:
I kept it because of https://github.com/widelands/widelands/pull/5937#issuecomment-1585531380

After the 1st tx pull ... so [there](https://github.com/widelands/widelands/pull/6064/files#diff-3712968fb6773d1263ad66dbf48a0eff788ce700d64e936497b2f64c5c7bb680R107) ... makes sense of course